.github/workflows/ci.yml

```diff
@@ -14,15 +14,10 @@ on:
 
 env:
   # Common versions
-  GO_VERSION: '1.23.1'
-  GOLANGCI_VERSION: 'v1.61.0'
+  GO_VERSION: '1.24.9'
+  GOLANGCI_VERSION: 'v2.4.0'
   DOCKER_BUILDX_VERSION: 'v0.24.0'
 
-  # These environment variables are important to the Crossplane CLI install.sh
-  # script. They determine what version it installs.
-  XP_CHANNEL: stable
-  XP_VERSION: v1.20.0
```

Comment:
Any reason we can't just use latest stable? I'm removed these env vars so that we just use latest stable by default.

This v1.20.0 version was causing push error messages like below because it still had `upbound.io` code in it, which was removed in https://github.com/crossplane/crossplane/pull/6562:
```
2025-10-21T00:43:13Z	DEBUG	Supplied server URL is not supported by this credentials helper	{"serverURL": "ghcr.io", "domain": "upbound.io"}
```